docs/install-and-run.md

```diff
@@ -254,7 +254,8 @@ steps in your setup:
     #import <EarlGrey/EarlGrey.h>
    ```
 
-2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/EarlGrey.swift) and add it to your test bundle. The file contains
+2. Download the correct [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/)
```

Comment:
But this only for non-pod flow. We need to make that clear otherwise you'd end up including earlgrey.swift twice.